packages/paste-core/components/combobox/__tests__/multiselect-combobox-customization.spec copy.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import type {RenderOptions} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import type {PasteCustomCSS} from '@twilio-paste/customization';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {Text} from '@twilio-paste/text';
+import {MultiselectCombobox} from '../src';
+
+const getStyles = (element = 'MULTISELECT_COMBOBOX'): {[key: string]: PasteCustomCSS} => ({
```

Comment:
Praise: I love this way of doing the customization tests!!!! Will follow this moving forward.